department-of-veterans-affairs/va.gov-cms
# VAGOV-4068: Migrate correct type for non-cta widgets.

To test:
1. Go to /admin/structure/migrate/manage/va_new_benefits/migrations/va_benefits_pension/execute and press 'Execute'.
2. When migration's done, go to /pension/eligibility and edit. 
3. Open the Q&A, "What should I do if I received an other than honorable, bad conduct, or dishonorable discharge?" and confirm that the React Widget has Widget Type, "pension-app-status".
